### ISSUE TITLE
Baseline vertically off-centered?

### ISSUE THREAD

**Issue by djmtype** [weiweihuanghuang/Prodigy-Sans#1]
Compared to Work Sans, Prodigy's baseline appears more off-centered vertically.

Same padding number, line-height set to same as type size, in Figma: https://cloudup.com/clEjdNaooTn

**Comment by weiweihuanghuang** [comment]
Good point, think the vertical metrics have been recalculated due to the yMin and yMax being changed... this is still according Google Webfonts scheme though
